plugins/BEdita/Core/src/Model/Table/ProfilesTable.php

```diff
@@ -38,6 +38,8 @@ public function initialize(array $config)
         $this->setPrimaryKey('id');
         $this->setDisplayField('name');
 
+        $this->addBehavior('BEdita/Core.Relations');
```

Comment:
@fquffio @batopa should this behavior be in `ObjectsTable` only? Or when should we add it on Table classes extending objects?